charts/kwok/templates/deployment.yaml

```
@@ -51,14 +52,7 @@ spec:
         - --enable-crds=ResourceUsage
         - --enable-crds=ClusterResourceUsage
         env:
-        - name: POD_IP
-          valueFrom:
-            fieldRef:
-              fieldPath: status.podIP
-        - name: HOST_IP
-          valueFrom:
-            fieldRef:
-              fieldPath: status.hostIP
+          {{ toYaml .Values.env | nindent 10 }}
```

Do the same for the other values to make them easier to understand.

```suggestion
        {{ with .Values.pod_ip }}
        - name: POD_IP
          value: {{ .Values.pod_ip }}
        {{ else }}
        - name: POD_IP
          valueFrom:
            fieldRef:
              fieldPath: status.podIP
        {{ end }}
        ...
```

---

I need to add additional environment variables into the pod